src/server/routes/api/v1/index.ts

```diff
@@ -33,5 +34,9 @@ export default function (
     prefix: 'parking',
   });
 
+  fastify.register(parkingSlot, {
+    prefix: 'parking',
+  });
+
```

Comment:
Both endpoints have the same `prefix`, this will cause conflicts when performing HTTP requests